src/Jabbr-Core.Data/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
```

Comment:
I'm going to make the necessary changes after the merge so that all references to models use the ones in the new .data package